tool/README.md

```diff
@@ -31,3 +31,245 @@ Debug logs found in `Settings > Copy Logs` are saved such that they can be read
 
 For a quick tutorial on how to navigate logs using `lnav`
 you can give [ their tutorial ](https://lnav.org/tutorials) a try.
+
+## Flutter versions that work with this instance of DevTools
```

Comment:
I would change this to
## Finding the Flutter version that a DevTools commit is compatible with

Because this readme update won't even be present if a user is looking at an old version of DevTools, so they should be reading these instructions on the master branch.